lib/node_modules/@stdlib/math/base/special/maxabsf/lib/index.js

```diff
@@ -26,8 +26,8 @@
 * @example
 * var maxabsf = require( '@stdlib/math/base/special/maxabsf' );
 *
-* var v = maxabsf( 3.14, -4.2 );
-* // returns ~4.2
+* var v = maxabsf( -4.0, 3.0 );
+* // returns ~4.0
```

Comment:
@gunjjoshi I don't think you need the `~` here or in any of the return value annotations.